src/__experimental__/components/date/date.stories.js

```diff
@@ -18,6 +19,15 @@ export default {
   },
 };
 
+i18n.changeLanguage("en-US");
+i18n.addResourceBundle("en-US", "carbon", {
+  date: {
+    formats: {
+      javascript: "M/D/YYYY",
```

Comment:
Has this short date format been browser tested? Sometimes a date without a leading zero can throw an error. It's a different format to that which you've added to the helper for the same `us` locale, nor is it in the `_defaultDateFormats` list.

Reply:
yes it works well, but probably I should remove it from a story.
![Zrzut ekranu 2021-03-5 o 15 24 11](https://user-images.githubusercontent.com/77274590/110128449-16d5b400-7dc7-11eb-97bf-81328da6e72b.png)